rpcs3/evdev_joystick_handler.cpp

```diff
@@ -182,11 +206,11 @@ int evdev_joystick_handler::scale_axis(int axis, int value)
             range.first = 0;
         }
 
-        return (static_cast<float>(value - range.first) / range.second) * 255;
+		return (static_cast<float>(Deadzone((((float)value - (float)range.first) / (float)range.second) * 255)));
```

Comment:
Not sure why the `static_cast<float>` for the return value of Deadzone, I think it would be better if Deadzone just returned a float.

Reply:
To add to what was said above, you should only have `static_cast<float>(value)`; all the others casts can be dropped.